bin/build.sh

```diff
@@ -5,10 +5,189 @@
 # PORT_ROOT must be defined to the root directory of the cloned ZOSOpenTools port 
 # Either PORT_TARBALL or PORT_GIT must be defined (but not both). This indicates where to pull source from
 # 
-# A certificate of the form: tarball.pem or git.pem must be in the PORT_ROOT directory 
 # Each dependent tool will have it's corresponding environment set up by sourcing .env from the installation
 # directory. The .env will be searched for in $HOME/zot/prod/<tool>, /usr/zot/<tool>, $HOME/zot/boot/<tool>
 
+checkdeps() {
+	deps=$*
+	fail=false
+	for dep in $deps; do
+		if ! [ -f "${HOME}/zot/prod/${dep}/.env" ] && ! [ -f "${HOME}/zot/boot/${dep}/.env" ] && ! [ -f "/usr/zot/${dep}/.env" ] ; then
```

Comment:
Are you adding /usr/zot/dev as an option for search?

Reply:
Should we use -r to make sure it's readable?

Reply:
after discussing with @IgorTodorovskiIBM we will just have /usr/bin/zot for now (which is a 'prod' build)